cli/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "xstate-cli",
+  "version": "0.0.1",
+  "bin": {
+    "xstate": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "esbuild src/bin.ts --outfile=bin/index.js --bundle --platform=node --target=node14 --external:fsevents"
```

Perhaps this would work:
```suggestion
    "prepare": "esbuild src/**/*.ts --outdir=bin --platform=node --target=node14"
```